container-deployment/docker-logging-driver.md

```diff
@@ -65,26 +65,24 @@ On successful start, you should see the Fluentd startup logs:
     @type stdout
   </match>
 </ROOT>
-2019-08-21 00:51:02 +0000 [info]: starting fluentd-1.3.2 pid=6 ruby="2.5.2"
-2019-08-21 00:51:02 +0000 [info]: spawn command to main:  cmdline=["/usr/bin/ruby", "-Eascii-8bit:ascii-8bit", "/usr/bin/fluentd", "-c", "/fluentd/etc/demo.conf", "-p", "/fluentd/plugins", "--under-supervisor"]
-2019-08-21 00:51:02 +0000 [info]: gem 'fluentd' version '1.3.2'
-2019-08-21 00:51:02 +0000 [info]: adding match pattern="*" type="stdout"
-2019-08-21 00:51:02 +0000 [info]: adding source type="forward"
-2019-08-21 00:51:02 +0000 [info]: #0 starting fluentd worker pid=16 ppid=6 worker=0
-2019-08-21 00:51:02 +0000 [info]: #0 listening port port=24224 bind="0.0.0.0"
-2019-08-21 00:51:02 +0000 [info]: #0 fluentd worker is now running worker=0
+2025-02-04 07:52:26 +0000 [info]: starting fluentd-1.16.7 pid=7 ruby="3.2.6"
+2025-02-04 07:52:26 +0000 [info]: spawn command to main:  cmdline=["/usr/local/bin/ruby", "-Eascii-8bit:ascii-8bit", "/usr/local/bundle/bin/fluentd", "--config", "/fluentd/etc/demo.conf", "--plugin", "/fluentd/plugins", "--under-supervisor"]
+2025-02-04 07:52:26 +0000 [info]: #0 init worker0 logger path=nil rotate_age=nil rotate_size=nil
+2025-02-04 07:52:26 +0000 [info]: adding match pattern="*" type="stdout"
+2025-02-04 07:52:26 +0000 [info]: adding source type="forward"
+2025-02-04 07:52:26 +0000 [info]: #0 starting fluentd worker pid=16 ppid=7 worker=0
+2025-02-04 07:52:26 +0000 [info]: #0 listening port port=24224 bind="0.0.0.0"
+2025-02-04 07:52:26 +0000 [info]: #0 fluentd worker is now running worker=0
 ```
 
 ### Step 3: Start Docker Container with Fluentd Driver
 
 By default, the Fluentd logging driver will try to find a local Fluentd instance \(Step \# 2\) listening for connections on the TCP port `24224`. Note that the container will not start if it cannot connect to the Fluentd instance.
 
-![Integration of Docker with Fluentd](https://www.fluentd.org/assets/img/recipes/fluentd_docker_integrated.png)
-
 The following command will run a base Ubuntu container and print some messages to the standard output:
 
 ```text
-$ docker run --log-driver=fluentd ubuntu echo "Hello Fluentd!"
+$ docker run --log-driver=fluentd ubuntu echo "Hello Fluentd\!"
```

Comment:
`echo "Hello Fluentd!"` does not works. It needs escape.

Reply:
How about just removing '!' from this example?
Using `!` is not important.